packages/cardano-graphql-services/src/TxSubmit/TxSubmitHttpServer.ts

```diff
@@ -67,7 +58,11 @@ export class TxSubmitHttpServer extends HttpServer {
       }
       res.send(body);
     });
-    return new TxSubmitHttpServer(config, router, { logger, txSubmitProvider });
+    return new TxSubmitHttpServer(
+      { logger, txSubmitProvider },
+      { router, healthCheckCallback: () => txSubmitProvider.healthCheck(), logger },
```

Comment:
Lint warning here. Let's configure it to fail on warnings. appending `--max-warnings 0` to root package.json lint script should do it